sdk/storage/Azure.Storage.Blobs/src/Models/BlobGetPropertiesOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+namespace Azure.Storage.Blobs.Models
+{
+    /// <summary>
+    /// Optional parameters for getting properties for a blob.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class BlobGetPropertiesOptions
```

Comment:
This class needs to include `BlobRequestConditions`

Reply:
We should also not mark this class `[EditorBrowsable(EditorBrowsableState.Never)]`